comp/dogstatsd/statsd/statsd.go

```diff
@@ -10,6 +10,7 @@ import (
 	"os"
 	"strconv"
```

Comment:
This file should be owned by @DataDog/agent-metrics-logs and the code owner should be updated [here](https://github.com/DataDog/datadog-agent/blob/3b86bc1c8fa7df4143c6613dab461ed2e484c0ec/.github/CODEOWNERS#L241).

Reply:
Should it ? This is the statsd client used by all clients, it's a "shared component" it doesn't have much to do with the server side of things (at least that is what we said last time I think)

@carlosroman what's your take on that ?